index.html

```diff
@@ -47,6 +47,113 @@ <h4>
                 <img src="assets/images/Disabled.svg" alt="">
             </div>
         </section>
+        <section id="my-recent-work">
+            <h2 class="title 2">My Recent<br>Work<br></h2>
+            <div class="indicator"></div>
+            <div class="img-patholder">
+                <div class="img-box"></div>

```

Comment:
Inside the `div` tags with the class of `img-box`, we should have an image placeholder. So an `img` tag with a plain image that should keep the place of our project image.

- [ ] please make sure to implement the `img` tag inside your `div` with the class of `img-box`.